pkgs/servers/nosql/mongodb/v5_0.nix

```
@@ -0,0 +1,18 @@
+{ stdenv, callPackage, lib, sasl, boost, Security, CoreFoundation, cctools }:
+
+let
+  buildMongoDB = callPackage ./mongodb.nix {
+    inherit sasl;
+    inherit boost;
+    inherit Security;
+    inherit CoreFoundation;
+    inherit cctools;
```

```suggestion
    inherit sasl boost Security CoreFoundation cctools;
```
this could be simplified to this.